CHANGELOG/CHANGELOG-1.15.md

```diff
@@ -0,0 +1,77 @@
+# Changelog 1.15
+
+## 1.15.0 - 2024-01-26
+
+### Added
+
+- Add a new feature `NodeNetworkPolicy` to allow user to apply `ClusterNetworkPolicy` to Kubernetes Nodes. ([#5658](https://github.com/antrea-io/antrea/pull/5658) [#5716](https://github.com/antrea-io/antrea/pull/5716), [@hongliangl] [@Atish-iaf])
```

Comment:
Can we add doc links for the well-documented features? This may help users get started quickly。
I think the first 4 are well documented.

Reply:
Done.